drivers/hmis_data_quality_tool/app/views/hmis_data_quality_tool/warehouse_reports/reports/_by_category.haml

```diff
@@ -1,18 +1,19 @@
 = render 'summary'
 - @report.results.group_by(&:category).each do |category, results|
-  %h2.mt-2= category
-  .card.mb-6
-    %table.table.table-striped
-      %thead
-        %tr
-          %th Section
-          %th Description
-          %th Required for
-          %th Overall Number in Universe
-          %th Number of Invalid Values
-          %th Percent Valid
-      %tbody
-        - results.each do |result|
-          = render 'row', result: result
+  %div{style: 'page-break-inside: avoid;'}
```

Comment:
```suggestion
  .perf-overview-breakdown
```
That's coming from the layout file: `app/views/layouts/performance_report.haml`

Reply:
Ugh, just realized that's a totally random class name, probably worth adding something like `.page-break-inside-avoid` in that declaration for the future.